pkg/istio-agent/grpcxds/grpc_bootstrap.go

```diff
@@ -31,13 +31,23 @@ import (
 	"istio.io/istio/pkg/file"
 )
 
+const (
+	ServerListenerNamePrefix = "lds/inbound|"
```

Comment:
Is this a new pattern ? This is turning a bit into an API - the 'lds/' and inbound| are a bit strange. Could we have a more intuitive/natural naming ?

Reply:
Something like `xds.istio.io/listener/inbound/%s`?